src/logger/logger.ts

```diff
@@ -9,13 +11,44 @@ const levels = {
     debug: 4,
 }
 
-const transports = [
-    new winston.transports.Console(),
-]
+interface Request extends IncomingMessage {
+    body: {
+        query: String;
+    };
+}
+
+const { combine, timestamp, printf } = format;
+
+/* 
+Possible bug here, returning: Property 'timestamp' does not exist on type 'TransformableInfo'
+when running tsc
+Need to add timestamp to the interface here: https://github.com/winstonjs/logform/blob/master/index.d.ts#L8
+*/
+// @ts-ignore
+const logFormat = printf(({ timestamp, level, message }) => {
+    return `${timestamp} ${level}: ${message}`;
+})
+
+export const logRequestMiddleware = (req: Request, res: Response, next: NextFunction) => {
+    if (req.method == 'OPTIONS') {
+        next()
+    } else if (req.body !== undefined && req.body.query !== undefined && req.body.query.indexOf('IntrospectionQuery') !== -1) {
+        next()
+    } else if (req !== undefined && req.body !== undefined) {
+        Logger.http(`Supergraph query: ${req.body.query}`)
```

Comment:
What does Logger.http() do? What format the message is logged int?

Reply:
This is explicit telling that the application is receiving an HTTP request, in this case, we track requests that contains the SQL query in the body.